packages/eslint-plugin-khan/lib/util.js

```diff
@@ -1,6 +1,7 @@
-const {execFile} = require("child_process");
+const {execFile, execSync} = require("child_process");
```

Comment:
It seems like using types would've solved this specific issue, as well, right?

Reply:
That too.  This code needs a lot of TLC.